data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -721,20 +404,20 @@
     "items": [
```

Comment:
The milspec and police spawn groups could use the same treatment of breaking things down by calibre.